src/components/common/ItemValues/GraphItemValue.module.css

```diff
@@ -1,17 +1,16 @@
 @import "@folio/stripes-components/lib/variables";
 
+.nodeCard:hover,
```

Comment:
I would note that for [UIWRKFLOW-35](https://folio-org.atlassian.net/browse/UIWRKFLOW-35):
> The selected state styles should not change on hover. Selection should override hover.

This implementation doesn't overide.
It uses the same state for both.

Reply:
Noted.